Dockerfile

```
@@ -17,6 +17,7 @@ CMD ["make", "test"]
 
 FROM gcr.io/distroless/base-debian10 as vvgo
 COPY --from=builder vvgo vvgo
+COPY ./etc /etc
 COPY ./public /public
```

```suggestion
COPY ./data/mime.types /etc/
COPY ./public /public
COPY --from=builder vvgo vvgo
```
We should probably keep all of these assorted files to the `data` tree.

This also reorders the `COPY`s for better caching.

---

Right now, everything in `./data` directory gets globbed up and built into the binary, so I don't want to put it in `./data`